docs/Playing/how-to-play.rst

```diff
@@ -1,224 +1,366 @@
+First of all, this isn't a *perfect* strategy, but it will get you started playing Freeciv21. Part of the
+appeal of Freeciv21 is in developing new strategies as you play more games. This guide is mostly based upon
+the Classic ruleset. There are other rulesets out there that ship with the game, but the base information
+presented here is the same for all rulesets.
+
+The game is divided into several steps or stages:
+
+#. Choosing the first city site.
+#. Selecting your first city locations.
+#. Selecting your technological advance.
+#. Mapping the countryside.
+#. Defending the cities.
+#. Deciding which units to build first.
+#. Improving the land.
+#. Deciding where to build cities.
+#. Taking care of the cities.
+#. Interacting with other players (diplomacy).
+#. Exploring the world.
+#. Things to keep in mind.
+#. Production phase.
+#. Annihilation of your opponents.
+
+Choosing the First City Site
+----------------------------
+
+Start the game by wandering around :strong:`briefly` to find a good place to build the first city. Do not feel
+tempted to investigate any yellow roofed villages yet, they might contain :unit:`Barbarian` tribes. Build the
+city close to Resources (fish, wheat, wine, or buffalo) and perhaps close to fresh water (River or Lake). Keep
+in mind that the first city you build will be your capital! It is advised not to place your capital on the
+coast. That can open you up to losing it from an attack from the sea.
+
+The idea is to balance the quality of the site you find against getting your first city established as early
+as possible. You will find that *balance* becomes a key aspect of playing Freeciv21.
+
+Selecting your First City Locations
+-----------------------------------
+
+In the Classic ruleset, you start the game with 5 :unit:`Settlers`, 6 :unit:`Workers`, and 2
+:unit:`Explorers`. Once you have planted your first city as your capital, you need to find a place for your
+remaining :unit:`Settlers` as quickly as possible. This initial expansion phase of the game is important to
+establishing your base cities. Cities are everything in Freeciv21. The more cities you have the more of every
+type of resource you can produce. Winning in Freeciv21 is a factor of your nation's ability to out produce
+your opponents.
+
+As mentioned in `Choosing the First City Site`_ above, you want to try and place your :unit:`Settlers` so they
+will create cities that have Resources available to them. These tile enhancements give bonuses to help your
+nation grow. In the game help under :title-reference:`Terrain --> Resources` you will find more
+information. Another thing you want to look for is sources of fresh water (River and Lake). Placing a city on
+or adjacent to sources of fresh water allow you to build a cheap :improvement:`Aqueduct/River` or
+:improvement:`Aqueduct/Lake` city improvements. An :improvement:`Aqueduct` allows your cities to grow larger
```

Comment:
The Classic ruleset has neither "Aqueduct/River" nor "Aqueduct/Lake". That's a LTT/LTX thing

Reply:
Wow, I whiffed on that one. I'll reword.

Reply:
I'm actually pulling the last part of that paragraph out.